src/main/java/com/owncloud/android/ui/activity/ScanDocActivity.java

```diff
@@ -0,0 +1,418 @@
+import android.view.View;
+import android.view.ViewGroup;
+import android.view.Window;
+import android.view.WindowManager;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.ImageView;
+import android.widget.ProgressBar;
+import android.widget.TextView;
+
+import com.owncloud.android.R;
+import com.owncloud.android.databinding.EditBoxDialogBinding;
+import com.owncloud.android.lib.common.utils.Log_OC;
+import com.owncloud.android.lib.resources.files.FileUtils;
+import com.owncloud.android.ui.adapter.ScanDocumentAdapter;
+import com.owncloud.android.ui.fragment.ScanDocumentFragment;
+import com.owncloud.android.ui.helpers.FileOperationsHelper;
+import com.owncloud.android.utils.BitmapUtils;
+import com.owncloud.android.utils.DisplayUtils;
+import com.owncloud.android.utils.ThemeUtils;
+
+import org.lukhnos.nnio.file.Files;
+import org.lukhnos.nnio.file.Paths;
+
+import java.io.IOException;
+
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AlertDialog;
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.constraintlayout.widget.ConstraintLayout;
+import androidx.viewpager2.widget.ViewPager2;
+import butterknife.BindView;
+import butterknife.ButterKnife;
+import butterknife.OnClick;
+import butterknife.Unbinder;
+
+import static com.owncloud.android.ui.activity.FileActivity.REQUEST_CODE__LAST_SHARED;
+
+public class ScanDocActivity extends AppCompatActivity implements ScanDocumentFragment.OnProcessImage {
+
+    public static final int REQUEST_CODE__TAKE_PICTURE_FROM_CAMERA = REQUEST_CODE__LAST_SHARED + 1;
+    public static final int REQUEST_CODE__RETAKE_PICTURE_FROM_CAMERA = REQUEST_CODE__LAST_SHARED + 2;
+    public static final String SCAN_DOC_ACTIVITY_RESULT_PDFNAME = "SCAN_DOC_ACTIVITY_RESULT_PDFNAME";
+
+    @BindView(R.id.progressBarScanDocument)
+    ProgressBar mProgressBar;
+
+    @BindView(R.id.ivAddAnOtherScanToDoc)
+    ImageView mBtnAddAnOtherScanToDoc;
+
+    @BindView(R.id.textViewPageCounter)
+    TextView mTextViewPageCounter;
+
+    @BindView(R.id.viewPagerScanDocument)
+    ViewPager2 mViewPagerScanDocument;
+
+    @BindView(R.id.btnClose)
+    Button mBtnClose;
+
+    @BindView(R.id.btnValidate)
+    Button mBtnValidate;
+
+    @BindView(R.id.ivNextScanDoc)
+    ImageView mImageViewNextScanDoc;
+
+    @BindView(R.id.contraintLayoutMainContainer)
+    ConstraintLayout mConstraintLayoutMainContainer;
+
+    @BindView(R.id.constraintLayout_crop_button)
+    ConstraintLayout mConstraintLayoutCropButton;
+
+    @BindView(R.id.constraintLayout_scan_doc_buttons)
+    ConstraintLayout mConstraintLayoutScanDocButtons;
+
+    @BindView(R.id.constraintLayout_pager_button)
+    ConstraintLayout mConstraintLayoutPagerButton;
+
+    @BindView(R.id.ivPreviousScanDoc)
+    ImageView mImageViewPreviousScanDoc;
+
+    int mCurrentPosition;
+    private Unbinder mUnbinder;
+    private String mPdfName = FileOperationsHelper.getScanDocName();
```

Comment:
```suggestion
    private String pdfName = FileOperationsHelper.getScanDocName();
```

We don't use the prefixes in new implementations anymore and refactor it when touching legacy code

Reply:
I don't understand, what should i do or use ?